tools/ansible/roles/dockerfile/files/settings.py

```diff
@@ -55,6 +55,9 @@ def get_secret():
 LOGGING['loggers']['awx']['handlers'] = ['console', 'external_logger']
 LOGGING['loggers']['awx.main.commands.run_callback_receiver']['handlers'] = ['console']
 LOGGING['loggers']['awx.main.tasks']['handlers'] = ['console', 'external_logger']
+
+# not sure how to change this
```

Comment:
I don't envision any changes to the `LOGGING` definition, because `awx.main.tasks` will still be the parent logger of the new python modules. Let me know if anything is unclear.